generateData.ts

```diff
@@ -0,0 +1,52 @@
+class Row {
```

Comment:
Please add build instructions for what you need to do to create the appropraite JS